src/mg_decoration.cpp

```diff
@@ -338,16 +338,63 @@ DecoSchematic::DecoSchematic()
 }
 
 
-size_t DecoSchematic::generate(MMVManip *vm, PcgRandom *pr, v3s16 p)
+bool DecoSchematic::canPlaceSchematicDecoration(MMVManip *vm, v3s16 p)
```

Comment:
Am I missing something or is this function identical to canPlaceSimpleDecoration?